client/packages/system/src/Patient/DetailModal/DetailModal.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { FC, useEffect } from 'react';
+import {
+  useTranslation,
+  useBreadcrumbs,
+  DetailContainer,
+  DetailInputWithLabelRow,
+  DetailSection,
+  Checkbox,
+  Grid,
+  useFormatDateTime,
+  Typography,
+  Box,
+  BasicSpinner,
+  MuiLink,
+} from '@openmsupply-client/common';
+import { useName } from '../api';
+
+interface DetailModalProps {
+  nameId: string;
+}
+
+export const DetailModal: FC<DetailModalProps> = ({ nameId }) => {
+  const { data, isLoading } = useName.document.get(nameId);
+  const t = useTranslation('common');
+  const { setSuffix } = useBreadcrumbs();
+  const isDisabled = true;
+  const { localisedDate } = useFormatDateTime();
+
+  useEffect(() => {
+    setSuffix(data?.name ?? '');
+  }, [data]);
+
+  if (isLoading) return <BasicSpinner />;
+
+  return !!data ? (
+    <DetailContainer>
+      <Box display="flex" flexDirection="column" alignItems="center" gap={2}>
+        <Typography sx={{ fontSize: 18, fontWeight: 700 }}>
+          {data.name}
+        </Typography>
+        <Grid container flex={1} flexDirection="row" gap={4}>
+          <DetailSection title="">
+            <DetailInputWithLabelRow
+              label={t('label.address')}
+              inputProps={{ value: data?.address, disabled: isDisabled }}
```

Comment:
I fleshed out the name sync a bit and broke this code while doing so :(

There is now address1 and address2 (was billingAddress in mSupply)...